src/test/resources/data/AllAccessControlUiTest.json

```diff
@@ -160,7 +160,7 @@
             "courseId":"idOfTypicalCourse1.access",
             "name":"Helper Course1",
             "email":"helper@course1.tmt",
-            "role":"Helper",
+            "role":"Custom",
             "isDisplayedToStudents":false,
             "displayedName":"Helper",
             "instructorPrivilegesAsText": "{\n  \"courseLevel\": {\n    \"canviewstudentinsection\": false,\n    \"cangivecommentinsection\": false,\n    \"cansubmitsessioninsection\": false,\n    \"canmodifysessioncommentinsection\": false,\n    \"canmodifycommentinsection\": false,\n    \"canmodifycourse\": false,\n    \"canviewsessioninsection\": false,\n    \"canmodifysession\": false,\n    \"canviewcommentinsection\": false,\n    \"canmodifystudent\": false,\n    \"canmodifyinstructor\": false\n  },\n  \"sectionLevel\": {},\n  \"sessionLevel\": {}\n}"
```

Comment:
Go ahead. BTW, I don't think it is optimal to store this data as a string inside json. Json can hold a better data structure or even an object containing this data. If we have a class that holds instructor privileges, we can simply convert that into a json. That is a separate issue though.